bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYZprime/run_card.template

```diff
@@ -0,0 +1,251 @@
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  $NAME     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  10000 = nevents ! Number of unweighted events requested 
+  $SEED   = iseed   ! rnd seed (0=assigned automatically=default))
```

Comment:
3000 = nevents
0 = iseed

should be good, the nevents 3000 is enough and iseed does not have to be randomly shaken for every single gridpacks

Reply:
Done